suse2013/common/l10n/en.xml

```diff
@@ -43,6 +43,9 @@
    <l:gentext key="Version" text="Version"/>
 
    <l:gentext key="step.optional" text="(Optional)"/>
+ 
+   <l:gentext key="tasksummary" text="Summary"/>
+   <l:gentext key="taskprerequisites" text="Requirements"/>
```

Comment:
hm, looking at https://github.com/docbook/xslt10-stylesheets/blob/master/gentext/locale/en.xml#L350 , I wonder: are the tasksummary/taskprerequisite/taskrelated elements supposed to have actual titles instead of something autogenerated? does geekodoc allow titles for these elements?

also, (stating the obvious)make sure to add some more languages...

Reply:
> I wonder: are the tasksummary/taskprerequisite/taskrelated elements supposed to have actual titles instead of something autogenerated? Does geekodoc allow titles for these elements?

Currently, no `title` inside of these `task*` elements are allowed in Geekodoc. So yes, that's a feature and not a bug. 😉 

Sure, we could allow titles, but I think in that case we would only get further inconsistencies which would "weaken" our task appearance. At the moment, I can't think of a use case where a title in these `task*` elements is absolutely necessary.

To summarize it, I would like to keep it that way for the time being. Let our writers gain some experiences. If we think we need it, we can add titles in a later version without any compatibility problems.